packages/integrations/cloudflare/README.md

```diff
@@ -357,6 +357,10 @@ import { Buffer } from 'node:buffer';
 
 Additionally, you'll need to enable the Compatibility Flag in Cloudflare. The configuration for this flag may vary based on where you deploy your Astro site. For detailed guidance, please refer to the [Cloudflare documentation on enabling Node.js compatibility](https://developers.cloudflare.com/workers/runtime-apis/nodejs).
 
+## Cloudflare module support
+
+All Cloudflare namespaced packages (like `cloudflare:sockets`) are whitelisted for use. Note that the package `cloudflare:sockets` does not work locally without using Wrangler dev mode.
```

Comment:
```suggestion
All Cloudflare namespaced packages (e.g. `cloudflare:sockets`) are allowlisted for use. Note that the package `cloudflare:sockets` does not work locally without using Wrangler dev mode.
```
So, it's mostly considered unacceptable to use the terms "whitelist" and "blacklist" now. I'm happy with any of the common alternatives for now, though admittedly, Astro hasn't standardized on one. We can just choose any acceptable alternative for now, and update if we do make a standard decision! 

"allowlist"  (vs "denylist") is the option I see most commonly used/recommended